python/src/index.rs

```diff
@@ -98,6 +98,13 @@ pub struct IndexConfig {
     pub columns: Vec<String>,
 }
 
+#[pymethods]
+impl IndexConfig {
+    pub fn __repr__(&self) -> String {
+        format!("Index({}, {:?})", self.index_type, self.columns)
```

Comment:
The first argument is probably clear, but the second argument is an arbitrary list of strings, so would be clearer what they mean if we added a name to them.
```suggestion
        format!("Index({}, columns={:?})", self.index_type, self.columns)
```